azure-pipelines/build.yml

```diff
@@ -281,13 +293,22 @@ steps:
     ArtifactName: 'logs'
   condition: true
 
-- task: NuGetCommand@2
+- task: DotNetCoreCLI@2
   displayName: 'NuGet push'
   condition: eq(variables['PUSH_SYMBOLS'], 'true')
   inputs:
     command: push
     packagesToPush: '$(Build.SourcesDirectory)/artifacts/nugetPackages/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
-    publishVstsFeed: '2191dd5f-4aec-491b-ac50-568bbc331c8a'
```

Comment:
What does this change for existing packages being published? Or is it just a task switch?

Reply:
Restored the task to original state.